src/components/dashboard/main-content/course-enrollments/course-cards/BaseCourseCard.jsx

```diff
@@ -456,15 +488,9 @@ const BaseCourseCard = ({
     );
   };
 
-  const renderChildren = () => {
-    if (children) {
-      return (
-        <Row>
-          <Col>
-            {children}
-          </Col>
-        </Row>
-      );
+  const renderCourseUpgradePrice = () => {
+    if (courseUpgradePrice) {
+      return <> {courseUpgradePrice} </>;
```

Comment:
nit: are the spaces before/after the `{}` needed?

Reply:
Updated to div to avoid linting error 👍🏽

Reply:
UPDATE: returned just the courseUpgradePrice instead 👍🏽